packages/components/inputs/radio-input/src/radio-group.tsx

```diff
@@ -11,11 +17,55 @@ const directionWrapper = {
   inline: Inline,
 };
 
-const Group = (props) => {
+export type TGroupProps = {
+  id?: string;
+  name?: string;
+  value: string | boolean;
+  onChange?: ChangeEventHandler;
+  onBlur?: FocusEventHandler;
+  onFocus?: FocusEventHandler;
+  isDisabled?: boolean;
+  isReadOnly?: boolean;
+  hasError?: boolean;
+  hasWarning?: boolean;
+  horizontalConstraint?:
+    | 7
+    | 8
+    | 9
+    | 10
+    | 11
+    | 12
+    | 13
+    | 14
+    | 15
+    | 16
+    | 'scale'
+    | 'auto';
+  direction?: keyof typeof directionWrapper;
+  directionProps?: object;
+  children: ReactElement | ReactElement[];
```

Comment:
Usually we use `ReactNode`. Any reason for not using that?

Reply:
This is because we need to clone the element and pass some props to it. 
ReactNode could be null, undefined, boolean etc which will not work for our use case. 
See line 86.

Reply:
React has a function `isValidElement`. I think we can use that to validate if the prop is an element.

https://reactjs.org/docs/react-api.html#isvalidelement

Reply:
Okay, that makes sense. 
Updated